src/server/.eslintrc

```diff
@@ -4,7 +4,8 @@
   },
 
   "rules": {
-    "strict": [2, "never"] // Not sure about it.
+    "strict": [2, "never"], // Not sure about it.
+    "no-console": true
```

Comment:
This actually works better for the whole server in my opinion than inline comments.